frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/tables/LogTable.java

```diff
@@ -172,9 +184,55 @@ public String generateDescription(Component source, Object itemId, Object proper
         table.setFilterBarVisible(true);
         table.setPageLength(pageLenght);
 
+        Button download = new Button(Messages.getString("DebuggingView.download"));
+        FileDownloader fileDownloader = new OnDemandFileDownloader(new OnDemandStreamResource() {
+            @Override
+            public String getFilename() {
+                return "log.csv";
+            }
+
+            @Override
+            public InputStream getStream() {
+
+                File fileToDownload;
+                try {
+                    fileToDownload = File.createTempFile("tempfile", "temp");
```

Comment:
The name for the temp file should be generated. If two users of UnifiedViews download logs at the same time, it will not behave correctly (because the same name for temp file is used)